seed/challenges/01-responsive-web-design/applied-visual-design.json

```diff
@@ -440,7 +441,7 @@
         "  <div class=\"cardContent\">",
         "    <div class=\"cardText\">",
         "      <strong><h4>Google</h4></strong>",
```

Comment:
This `<strong>` tag has no effect since `h4`s are bold by default. Would you mind removing the `<strong>` tag for this and the following challenges now while you're at it? :blush: